doc/whats_new.rst

```diff
@@ -52,6 +52,7 @@ Detailed list of changes
 ^^^^^^^^^^^^
 
 - Amending a dataset now works in cases where the newly-written data contains additional participant properties (new columns in ``participants.tsv``) not found in the existing dataset, by `Richard Höchenberger`_ (:gh:`1113`)
+- Fixes copyfile_eeglab to prevent data int conversion and eeg_checkset failure, by `Laetitia Fesselier`_ (:gh:`1122`)
```

Comment:
```suggestion
- Fixes :func:`~mne_bids.copyfiles.copyfile_eeglab` to prevent data type conversion leading to an ``eeg_checkset`` failure when trying to load the file in EEGLAB, by `Laetitia Fesselier`_ (:gh:`1122`)
```